### ISSUE TITLE
Placeholder expansion registration initializing...

### ISSUE THREAD

**Issue by 3328429240** [PlaceholderAPI/PlaceholderAPI#1070]
### Confirmation

- [X] My issue isn't already found on the Issue tracker.
- [X] My issue is about **PlaceholderAPI** and not any expansion or external plugin
- [X] The issue isn't already fixed in a Spigot Release or Development Build.
- [x] The [Common Issues](https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Common-Issues) page doesn't mention this issue.

### Type

Plugin Bug

### What happens?

After the server starts, the variables can be displayed and output correctly, but a few seconds after the server starts, it displays' Placeholder expansion registration initialization... ' After the words were written, a command similar to 'papi reload' was executed, which reloaded all variables, preventing the previously registered variables from being used properly.
After this information, the 'guide' variable that previously displayed 'PlaceholderAPI] Successfully registered internal expansion: guide [1.0.0]' in the log does not appear in the 'papi list' list, and it does not parse properly when used. Therefore, I suggest adding a switch to optionally perform the reload like operation after server startup. Thank you—— Google Translate

### Expected Behaviour

At the prompt 'Placeholder expansion registration initialization...' The previously registered variables can still be used afterwards

### How to Reproduce

Start server

### `/papi dump` Output

none

**Comment by Andre601** [comment]
This is not an issue with PlaceholderAPI but with the Guide expansion. Let me explain.

Based on the info you gave is Guide an internal expansion, which means 2 things:
1. It's not a separate jar file you add to your expansions folder, but a class inside an existing plugin.
2. Said plugin is registering the expansion, not PlaceholderAPI.

And there is the issue. The plugin that contains the Guide expansion is registering it, not PlaceholderAPI. This comes with the downside, that during a reload of PlaceholderAPI, it will unregister all expansions, but only register those inside the expansions folder again, as those are the only ones it can know about (How should it know about other expansions in a plugin? It does not and cannot know what plugin registers the expansion).

That's why internal expansions **need** to override the `persist()` method and have it return `true`, as this tells PlaceholderAPI to not unload it during a reload, avoiding this issue.

So the issue is the fact that the Guide expansion is not overriding said persist method, making PlaceholderAPI unregister it during a reload. And since it is not an external expansion in the expansions folder can PlaceholderAPI not "know" about it and therefore not register it...

tl;dr: This is a configuration error with the Guide expansion and should be fixed on their end and not ours... Especially since this wouldn't be something that can be fixed by executing a command after server startup (That would frankly make it even worse).
